xcpmd/src/vm-actions-module.c

```diff
@@ -526,6 +527,10 @@ void shutdown_dependencies_of_vm(char * vm_path, char * type) {
         property_get_com_citrix_xenclient_xenmgr_vm_state_(xcdbus_conn, XENMGR_SERVICE, vm_identifier_table->entries[i].path, &state);
         deps_list_entry->vm_state = clone_string(state);
```

Comment:
Not g_free()'d, since [xcdbus](https://github.com/OpenXT/libxcdbus/blob/master/src/xcdbus.c#L870) creates an unboxed duplicate string, but yes, it does need to be freed.